ballerina/tests/redis_connection_command_test.bal

```diff
@@ -19,7 +19,7 @@ import ballerina/test;
 @test:Config {
 }
 function testPing() {
-    var result = conn->ping();
+    var result = redis->ping();
     if (result is string) {
         test:assertEquals(result, "PONG");
     } else {
```

Comment:
We can refactor the code like this:
```ballerina
function testPing() returns error? {
    string result = check redis->ping();
    test:assertEquals(result, "PONG");
}
```

Check for other places as well.

Reply:
I have already done this for the test cases which I have modified. Planning to send a separate PR with all the similar "unrelated" changes (i.e. removing parenthesis from if else clauses) for the entire test suite, as the intended changes might go unnoticed otherwise. Hope that is okay.